modules/oadp-configuring-dpa-multiple-bsl.adoc

```diff
@@ -7,7 +7,19 @@
 [id="oadp-configuring-dpa-multiple-bsl_{context}"]
 = Configuring the DPA with more than one BSL
 
-You can configure the DPA with more than one BSL and specify the credentials provided by the cloud provider.  
+You can configure the `DataProtectionApplication` (DPA) custom resource (CR) with more than one `BackupStorageLocation` (BSL) and specify the credentials provided by the cloud provider.
+
+[NOTE]
+====
+Consider the following scenario, where you have configured the following two BSLs:
+
+* Configured one BSL in the DPA and set it as the default BSL.
+* Created another BSL independently by using the `BackupStorageLocation` CR.
+
+Because you have already set the BSL created through the DPA as the default, you cannot set the independently created BSL again as default.
+
+This means, at any given time, you can set only one BSL as the default BSL.
+====
```

Comment:
```suggestion
[NOTE]
====
You can set only one of the two BSLs as default at a time. For example, if you set a BSL that is configured in the DPA as default, do not set the other BSL as default, which is independently configured by using the `BackupStorageLocation` CR.
====
```

As per my understanding, the note guides about making the either of BSLs a default one. IMO, the note can be summed up as suggested above by reducing some words as per the minimalism principle 4.